rc_django/decorators.py

```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.utils.module_loading import import_string
+from django.shortcuts import render
+
+
+def can_proxy_restclient(request, service, url):
```

Comment:
Should this throw an exception instead of a print statement?

Reply:
Say: https://docs.python.org/2/library/exceptions.html#exceptions.NotImplementedError

Reply:
Good suggestion!

We need to determine if we want to  have the default implementation.